requirements.txt

```diff
@@ -9,3 +9,4 @@ pylint
 libvirt-python
 pyvmomi
 packaging
+python-openstackclient
```

Comment:
I don't see where you use it.

Reply:
It used to be used everywhere and the code is missing requirements and documentation and basically most of the non-code work for OpenstackHost. I am just updating that and because it is one big commit without a commit message it is not clear. But it is laready used in `_run_openstack()`